api/src/main/java/org/openmrs/PatientProgram.java

```diff
@@ -57,6 +64,11 @@ public PatientProgram() {
 	public PatientProgram(Integer patientProgramId) {
 		setPatientProgramId(patientProgramId);
 	}
+
+	/**Constructor with Patient Program */
+	public PatientProgram(PatientProgram patientProgram) {
```

Comment:
Can you please remove this? They had it in Bahmini just because they wanted to construct their own instance from a PatientProgram so they can link them, you don't need it in core.